samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -518,6 +548,19 @@ CONFIG_LED_CONTINUOUS_INDICATION - Continuous LED status indication
 CONFIG_TEST_COUNTER - Enable test counter
    Enables the test counter.
 
+.. _CONFIG_AT_CMD_REQUESTS:
+
+CONFIG_AT_CMD_REQUESTS - Enable AT command requests
+   AT command requests allow remote execution of modem AT commands, requested using application-specific AT commands.
```

Comment:
This is very confusing. Can you simplify it? I am not at all sure how.

Can it possibly be something like: "Application-specific AT command requests allow..."

Reply:
Yeah, I think this sentence got garbled by accident

I think I meant to write:

> Allow remote execution of modem AT commands, requested using application-specific AT commands.

Do you think that would work, or does it still require simplification?

Reply:
Maybe that does the trick. If you think it reads well. You are the native speaker, after all.